### PR TITLE
[IOTDB-941] Support 'delete storage group <prefixPath>'

### PR DESCRIPTION
**Pull request by LebronAl**
https://issues.apache.org/jira/projects/IOTDB/issues/IOTDB-941

### CODE REVIEW

**Comment by LebronAl** [comment]
https://issues.apache.org/jira/projects/IOTDB/issues/IOTDB-941

File: server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java [hunk 1]
```diff
@@ -1055,13 +1058,19 @@ private boolean createTimeSeries(CreateTimeSeriesPlan createTimeSeriesPlan)
...
+          createMultiTimeSeriesPlan.getEncodings().get(i),
+          createMultiTimeSeriesPlan.getCompressors().get(i),
+          createMultiTimeSeriesPlan.getProps() == null ? null
+              : createMultiTimeSeriesPlan.getProps().get(i),
+          createMultiTimeSeriesPlan.getTags() == null ? null
+              : createMultiTimeSeriesPlan.getTags().get(i),
+          createMultiTimeSeriesPlan.getAttributes() == null ? null
+              : createMultiTimeSeriesPlan.getAttributes().get(i),
+          createMultiTimeSeriesPlan.getAlias() == null ? null
+              : createMultiTimeSeriesPlan.getAlias().get(i));
```
Comment (Alima777) [inline server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java:1073]: Recover the format...

**Review by Alima777 — CHANGES_REQUESTED** [review]

File: server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java (hunk 1 above)
Comment (LebronAl) [reply server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java:1073]: Actually I have imported `java-google-style.xml` in the root dir before formating this file.Maybe the old format is not suitable in `java-google-style.xml`. As you can see, the default hard warp is 100.
image
If you put them on one line, the code on the right will exceed the hard limit.
image

Comment (Alima777) [reply server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java:1073]: You can rename the `createMultiTimeSeriesPlan` to a shorter one. It's soooo ugly now.

Comment (LebronAl) [reply server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java]: Good suggestion!

**Review by jt2594838 — APPROVED** [review]

**Review by Alima777 — APPROVED** [review]

**Pull request merged**